app/src/main/java/com/strv/movies/MainActivity.kt

```diff
@@ -46,13 +44,94 @@ class MainActivity : ComponentActivity() {
                                 DarkLightModeSwitchIcon(isDarkTheme = isDarkTheme)
                             }
                         )
+                        Greeting("MOVIES")
+                        LogInMoviesApp()
                         MoviesList(movies = OfflineMoviesProvider.getMovies())
                     }
                 }
             }
         }
     }
 
+    @OptIn(ExperimentalFoundationApi::class)
+    @Composable
+    fun LogInMoviesApp() {
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier
+                .padding(start = 40.dp, bottom = 10.dp, top = 18.dp, end = 20.dp)
```

Comment:
You usually want the start and end padding to be the same so the content is centered horizontally.